### ISSUE TITLE
Support 2FA and ssh

### ISSUE THREAD

**Issue by cpritchett** [saiftynet/EZPWC#5]
Github accounts with 2FA don't seem to be supported.  Which would be fine, but the script currently also doesn't seem to support pre-configured GitHub ssh settings or keys.

**Comment by saiftynet** [comment]
Agreed, big issue....now this probably would be possible with the [Net::GitHub cpan module] (https://metacpan.org/pod/Net::GitHub). I do plan to do this in the future...BUT am concerned about accidentally exposing users credentials or storing these in a file. Currently EZPWC forces the user to interact via a backtick execution of git.  I am, as far as github programming is concerned, a newbie and am purely using stuff from manwar's instructions.

**Comment by saiftynet** [comment]
Hi Chad,
Have uploaded version 0.09 which tries to use credential.helper.  Uploading ssh key to github appears to be a manual process, once a suitable key is uploaded things may work automatically. Can you advise if this is the case, or how I could enable this feature.  The extra code is at line 177, `sub setupGithub2()`
